packages/terra-pills/tests/wdio/pills-spec.js

```diff
@@ -0,0 +1,118 @@
+const ignoredA11y = {
+  'aria-roles': { enabled: false },
+};
+
+Terra.describeViewports('Pills', ['medium'], () => {
+  describe('Single Selectable Removable Pills', () => {
+    it('display a selectable-pill', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/pills/filter-pills/single-selectable-removable-filter-pill');
+      Terra.validates.element('truncated', { rules: ignoredA11y });
+    });
+
+    it('focus the pill', () => {
+      browser.keys('Tab');
+      Terra.validates.element('focus the pill', { rules: ignoredA11y });
+    });
+
+    it('open popup', () => {
+      browser.keys('Enter');
```

Comment:
This is the key command for the pop-up, I see now. Please add some doc making this clear to consumers.